vipps-recurring-api-faq.md

```diff
@@ -40,9 +40,9 @@ Document version: 1.6.1.
 * [How can I change partners for a merchant?](#how-can-i-change-partners-for-a-merchant)
 * [At what time during the day are charges made?](#at-what-time-during-the-day-are-charges-made)
 * [How do I check my customer's status?](#how-do-i-check-my-customers-status)
-* [A customer's charge failed, but I did not receive any warning](#a-customers-charge-failed-but-i-did-not-receive-any-warning)
+* [A customer's charge failed, but I did not receive any warning](#a-customers-charge-failed-but-did-not-receive-any-warning)
 * [I don't want a charge to fail the first time the transaction fails (insufficient funds / networking issues etc.)](#i-dont-want-a-charge-to-fail-the-first-time-the-transaction-fails-insufficient-funds--networking-issues-etc)
-* [Can the charge for an Agreement be changed?](#can-the-charge-for-an-agreement-be-changed)
+* [Can the charge for an Agreement be changed?](#can-the-charge-be-different-from-the-agreements-price)
```

Comment:
Shoyldn't the ToC entry and the actual paragraph heading be the same?
```suggestion
* [Can the charge for an Agreement be different from the agreement price?](#can-the-charge-be-different-from-the-agreements-price)
```

Reply:
Yes they should, forgot to update them. Thanks !